src/rvpt/rvpt.h

```diff
@@ -7,7 +7,7 @@
 
 #include <vulkan/vulkan.h>
 
-#define GLFW_INCLUDE_VULKAN
+#define GLFW_INCLUDE_NONE
```

Comment:
Only adding for posterity that this change is fine, as GLFW_INCLUDE_VULKAN is redundant since this file manually includes vulkan.h. 
https://www.glfw.org/docs/3.3/build_guide.html#build_macros